cypress/integration/resource.spec.js

```diff
@@ -92,4 +92,27 @@ context('Actions', () => {
     cy.get('[data-testid=report-dialog-submit-button]').click();
     cy.get('[data-testid=report-dialog]').should('not.exist');
   });
+
+  it('is possible to request DOI', () => {
+    const publishedTestPost = mockMyResources[0];
+    cy.visit(`/resource/${publishedTestPost.identifier}]`);
+    cy.get('[data-testid=request-doi-button').click();
```

Comment:
fyi: vi har nettopp begynt å legge alle data-testider i en samlet fil, slik at vi slipper å skrive de flere plasser: https://github.com/BIBSYSDEV/NVA-Frontend/blob/4f7ae5ea297a93bfa892979ef8d18323870c97f2/src/utils/dataTestIds.ts

Reply:
Takk for tipset, noterer det ned så tar jeg det med Per når han er på plass igjen.